<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphMan</class>
 <widget class="QMainWindow" name="GraphMan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1258</width>
    <height>660</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1258</width>
    <height>660</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1258</width>
    <height>660</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Графы</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphTV">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>881</width>
      <height>641</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QPushButton" name="addVertexBTN">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Alternates</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Добавить вершину</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delVertexBTN">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Alternates</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Удалить вершину</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addEdgeBTN">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Alternates</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Добавить/изменить ребро</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delEdgeBTN">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Alternates</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Удалить ребро</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="addEdgeStart">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Alternates</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>18</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="addEdgeFinish">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>260</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Alternates</family>
     </font>
    </property>
    <property name="maxLength">
     <number>18</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="addEdgeWeight">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>260</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Alternates</family>
     </font>
    </property>
    <property name="maxLength">
     <number>18</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="addEdgeReverseWeight">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>260</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Alternates</family>
     </font>
    </property>
    <property name="maxLength">
     <number>18</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="delEdgeStart">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Alternates</family>
     </font>
    </property>
    <property name="maxLength">
     <number>18</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="delEdgeFinish">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>360</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Alternates</family>
     </font>
    </property>
    <property name="maxLength">
     <number>18</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="delVertexInput">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>160</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Alternates</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>18</number>
    </property>
   </widget>
   <widget class="QPushButton" name="updateTVBTN">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Alternates</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Обновить экран</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tableBTN">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Alternates</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Режим таблицы смежности</string>
    </property>
   </widget>
   <widget class="QPushButton" name="graphBTN">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Alternates</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Режим графа</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BFS_BTN">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Alternates</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Обход в ширину</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="BFS_start">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>410</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Alternates</family>
     </font>
    </property>
    <property name="maxLength">
     <number>18</number>
    </property>
   </widget>
   <widget class="QPushButton" name="DFS_BTN">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Alternates</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Обход в глубину</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="DFS_start">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>460</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Alternates</family>
     </font>
    </property>
    <property name="maxLength">
     <number>18</number>
    </property>
   </widget>
   <widget class="QPushButton" name="deicstraBTN">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Alternates</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Алгоритм Дейкстры</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="deicstraStart">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Alternates</family>
     </font>
    </property>
    <property name="maxLength">
     <number>18</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="deicstraFinish">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>560</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Alternates</family>
     </font>
    </property>
    <property name="maxLength">
     <number>18</number>
    </property>
   </widget>
   <widget class="QTextBrowser" name="output">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Montserrat Alternates</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
